### PR TITLE
New version: Tencent.WeChat version 3.9.5.90

### PR DESCRIPTION
**Pull request by SpecterShell**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111654&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/t/Tencent/WeChat/3.9.5.90/Tencent.WeChat.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.7-3-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Tencent.WeChat
+PackageVersion: 3.9.5.90
+PackageLocale: en-US
+Publisher: 腾讯科技(深圳)有限公司
```
Comment (CoolPlayLin) [inline manifests/t/Tencent/WeChat/3.9.5.90/Tencent.WeChat.locale.en-US.yaml:7]:
```suggestion
Publisher: Tencent Technology (Shenzhen) Co., Ltd
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]

File: manifests/t/Tencent/WeChat/3.9.5.90/Tencent.WeChat.locale.en-US.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/t/Tencent/WeChat/3.9.5.90/Tencent.WeChat.locale.en-US.yaml:7]: The `Publisher` field should match the ARP entry written by the installer. Although the installer can write different `DisplayName` values under different locales, it writes the same value for `Publisher` entry.
image

**Review by ItzLevvie — APPROVED** [review]

**Comment by BrandonWanHuanSheng** [comment]
@CoolPlayLin  Can you mark the conservation as resolved for the merge to take effect?

**Comment by CoolPlayLin** [comment]
I have deleted the change request

**Comment by CoolPlayLin** [comment]
May need @SpecterShell to process

**Pull request merged**